packages/webapp/src/_app/ui/header-nav.tsx

```diff
@@ -13,66 +12,49 @@ interface MenuItem {
-                <div className="relative flex items-center justify-between h-16">
-                    <div className="absolute inset-y-0 left-0 flex items-center sm:hidden">
-                        {/* <HeaderMobileButton
-                            open={mobileMenuOpen}
-                            handleMobileMenu={handleMobileMenu}
-                        /> */}
-                    </div>
-                    <div className="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start">
-                        <HeaderLogo />
-                        <HeaderItems menuItems={menuItems} />
-                    </div>
-                    <AccountMenu />
-                </div>
-            </div>
-            {/* <HeaderMobileMenu
-                open={mobileMenuOpen}
-                menuItems={menuItems}
-                handleNavigationClick={() => setMobileMenuOpen(false)}
-            /> */}
-        </nav>
-    );
-};
+export const HeaderNav = () => (
+    <header className="text-gray-600 body-font border-b border-gray-200 bg-white">
+        <div className="container mx-auto flex flex-wrap p-5 flex-col md:flex-row items-center md:h-20">
+            <HeaderLogo />
+            <nav className="md:mr-auto md:ml-4 md:py-1 md:pl-4 md:border-l md:border-gray-400	flex flex-wrap items-center text-base justify-center">
+                <HeaderItems menuItems={MENU_ITEMS} />
+            </nav>
+            <AccountMenu />
+        </div>
+    </header>
+);
 
 const HeaderLogo = () => (
-    <div className="flex-shrink-0 flex items-center">
-        <Link href="/">
-            <a className="text-2xl text-yellow-500 font-bold">EdenOS</a>
-        </Link>
-    </div>
+    <Link href="/">
+        <a className="flex title-font items-center mb-4 md:mb-0">
+            <span className="text-2xl text-yellow-500 font-bold">EdenOS</span>
+        </a>
+    </Link>
 );
 
 const HeaderItems = ({ menuItems }: { menuItems: MenuItem[] }) => {
     return (
-        <div className="hidden sm:block sm:ml-6">
-            <div className="flex space-x-4">
-                {menuItems.map((item, index) => (
-                    <HeaderItemLink
-                        key={index}
-                        href={item.href}
-                        exactPath={item.exactPath}
-                    >
-                        {item.label}
-                    </HeaderItemLink>
-                ))}
-            </div>
-        </div>
+        <>
+            {menuItems.map((item, index) => (
+                <HeaderItemLink
+                    key={index}
+                    href={item.href}
+                    exactPath={item.exactPath}
+                >
+                    {item.label}
+                </HeaderItemLink>
+            ))}
+        </>
     );
 };
 
 const HEADER_LINK_BASE_CLASS =
-    "block px-3 py-2 rounded-md text-base font-medium hover:bg-gray-700 hover:text-white ";
+    "mr-5 last:mr-0 hover:text-gray-900 ";
```

Comment:
I feel like you need to install prettier and run it in the files that you edited. since the .prettierrc line lengh has enough space, this should be in a single line: 
`const HEADER_LINK_BASE_CLASS = "mr-5 last:mr-0 hover:text-gray-900 ";`

Reply:
Ah, possibly. Will do.